test/provider_os_memory_multiple_numa_nodes.cpp

```diff
@@ -391,7 +417,16 @@ TEST_F(testNuma, checkModeInterleave) {
         EXPECT_NODE_EQ((char *)ptr + page_size * i, numa_nodes[index]);
     }
 
-    bitmask *retrieved_nodemask = retrieve_nodemask(ptr);
+    auto [fileLine, retrieved_nodemask] = retrieve_nodemask(ptr);
+    if (fileLine != 0) {
+        if (retrieved_nodemask == nullptr) {
+            GTEST_FAIL() << "retrieve_nodemask(): numa_allocate_nodemask() "
+                            "returned nullptr";
+        } else {
+            GTEST_FAIL() << "retrieve_nodemask(): get_mempolicy() return value "
```

Comment:
again, I think this is overcomplicated - what if we add another place, where `retrieve_nodemask` could fail? some generic message should be okay, without splitting it into two cases.

Reply:
Changed the text